src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  onClose: React.Dispatch<React.SetStateAction<Todo | null>>;
+  selectedTodo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({ onClose, selectedTodo }) => {
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [loading, setLoading] = useState(true);
```
to fix modal blinks when you try to open it